src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -1054,13 +1092,25 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 					
 					self?.submitExposure(showSubmissionSuccess: true, isLoading: isLoading)
 				case .srs:
-					self?.submitSRSExposure(showSubmissionSuccess: true, isLoading: isLoading)
+					self?.showSRSFlowConsentAlert(for: .confirmWarnOthers(on: exposureSubmissionSymptomsOnsetViewController), isLoading: isLoading)
 				case .none:
 					break
 				}
 			},
 			onDismiss: { [weak self] isLoading in
-				self?.showSubmissionSymptomsCancelAlert(isLoading: isLoading)
+				guard let self = self else { return }
```

Comment:
Same piece of code like above with just a change in enum type, could be wrapped in a function?